baggers.js

```diff
@@ -13299,6 +13299,42 @@ var data = {
         "twitter": "gallain",
         "mail": "guillaume@llain.fr"
       }
+    },
+    {
+      "since": "2016-07-18",
+      "name": "Frédéric Pot",
+      "bio": "Co-fondateur de Lucca en 2002, je suis le Product Owner de la solution Figgo utilisée par 115 000 salariés de 900 sociétés à travers 30 pays. Je serai ravi de vous partager mon expérience sur les 15 dernières années autour de l’évolution dans la gestion d’un logiciel RH.",
+      "picture": "https://avatars1.githubusercontent.com/u/2751422?v=3&u=93aa7a0037a926090eb975aaf590840a069f6a44&s=140",
+      "websites": [
+        {
+          "name": "Lucca",
```

Comment:
Name should conform with doc. Read https://github.com/brownbaglunch/bblfr_data/blob/gh-pages/README.md